src/components/Rating.jsx

```diff
@@ -72,6 +72,13 @@ export default class Rating extends Component {
   }
 
```

Comment:
I added this function here coz I found that for react components like <input> you can always query node's value to get real value of component.
So Rating / Select should also do that.